config/mysqlpersister/connector.go

```diff
@@ -0,0 +1,34 @@
+package mysqlpersister
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+type UnsafeConnector struct {
+	dbUser string
+	dbPass string
+	dbHost string
+	dbPort int
+	dbName string
+}
+
+func NewUnsafeConnector(dbUser, dbPass, dbHost string, dbPort int, dbName string) *UnsafeConnector {
```

Comment:
What is the purpose of having a `New` function if `UnsafeConnector` is exported?

Reply:
It's a pattern I've picked up?  Sort of a synthesis of `accept interfaces, return concrete types` and the fact that `golint` warns about unexported return types.